shortcut.go

```diff
@@ -144,3 +144,51 @@ func (se *ShortcutSelectAll) Mod() KeyModifier {
 func (se *ShortcutSelectAll) ShortcutName() string {
 	return "SelectAll"
 }
+
+// ShortcutUndo describes a shortcut undo action.
+type ShortcutUndo struct{}
```

Comment:
These new types need the extra:

```go
//
// Since: 2.5
```

Reply:
Ah yes. Didn't think of that either.

I added it.